client/dashboard/sites/performance/style.scss

```diff
@@ -0,0 +1,102 @@
+.site-performance-report {
+	--studio-white: #fff;
```

Comment:
@jasmussen Will we be including the color studio variables moving forward? I've bubbled them up here because they are used in the underlying package and we aren't enqueuing them in the new dashboard. Would it make sense to replace them with an equivalent?